src/pipx/util.py

```diff
@@ -55,7 +55,10 @@ def rmdir(path: Path, safe_rm: bool = True) -> None:
     logger.info(f"removing directory {path}")
     try:
         if WINDOWS:
-            os.system(f'rmdir /S /Q "{str(path)}"')
+            # The packaged app (Microsoft Store) version of Python uses path
+            # redirections, but `rmdir` won't follow these, so use realpath()
+            # to manually apply the redirection first.
+            os.system(f'rmdir /S /Q "{os.path.realpath(path)}"')
```

Comment:
use subprocess call instead

Reply:
Can't do that - `rmdir` is a Windows cmd shell command, it's not an executable.

I'll admit I have no idea why that code is using `rmdir` instead of `shutil.rmtree` on Windows, but that's besides the point of this PR - this discussion should be had elsewhere.

Reply:
The reviewer marks comments resolved when they are happy with the result. Why do we need shell command here and not just use Pythons rm?

Reply:
> The reviewer marks comments resolved when they are happy with the result.

Apologies. Where I'm from, the author marks comments as resolved and the reviewer should feel free to unresolve them if they disagree with the resolution.

> Why do we need shell command here and not just use Pythons rm?

I have no idea. I traced that code all the way back to 7cb6561c53ac338c0f9b134b042dcca3ccf49917 which was written by @cs01 5+ years ago and doesn't come with any detailed rationale.

I agree that using `shell.rmtree()` would be simpler and wouldn't have the path resolution problem (because the operation is done directly from the python process, which will apply the correct redirections).

The only reason why I didn't just change the code to use `shell.rmtree()` on Windows is because I wanted my PR to be as focused and unintrusive as possible. I just went back and added a commit to the PR to make the change; let me know if you want to keep that commit or not.

Reply:
You mean shutil.rmtree? I'd prefer that over calling system command 🤔

Reply:
Yes, sorry, I meant `shutil.rmtree()`. If you're happy with that, I updated the PR to use it. Feel free to review.